avocado/core/varianter.py

```diff
@@ -432,6 +434,6 @@ def itertests(self):
             for variant in iter(iter_variants):
                 yield variant
         else:   # No variants, use template
-            yield {"variant": self.default_params.get_leaves(),  # self.default_params is TreeNode pylint: disable=E1101
+            yield {"variant": self._default_params.get_leaves(),
```

Comment:
You asked on the previous version, so here is the answer again: please include (and squash) this one.